src/js/display-forecasts.js

```diff
@@ -15,15 +15,22 @@ export function displayToday(todayData) {
     todayDate.textContent = todayOnScreen;
   }
 
-  weatherIconsData.forEach((icon) => {
```

Comment:
you can export icon data as "iconName" : "imageURL" and access to image by iconName.
You should't iterate an array to find an icon.